lib/render.js

```diff
@@ -45,7 +46,7 @@ module.exports = function (html, options) {
     }
   }
 
-  var parser = MD(mdOptions).use(lazyHeaders).use(emoji, {shortcuts: {}})
+  var parser = MD(mdOptions).use(lazyHeaders).use(emoji, {shortcuts: {}}).use(expandTabs, {tabWidth: 4})
```

Comment:
am i about to argue about tab spacing?
i'm about to argue about tab spacing.

thoughts on 2 vs 4? i'm a 2.